### PR TITLE
Fix: Make samples of pools non-invoicable

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR fixes 
- so that samples created for pools defaults to non-invoicable in order to not double invoice pools
- so that FLUFFY pools can be handled as RML when submitted
- so that FLUFFY pools are validated by the same schema as RML

### How to prepare for test
- [x] ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [x] do submit an RML order in OP

### Expected test outcome
- [x] check that the pool has no_invoice == False and the samples has no_invoice == True
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by MM
- [x] tests executed by
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Deploy this branch
- [ ] Inform to VW, AZ, production, lab

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
Test result:
image
image
🆗 according to expected test outcome

**Review by moonso — APPROVED** [review]
👍

File: cg/meta/orders/api.py [hunk 1]
```diff
@@ -173,8 +173,15 @@ def _add_customer_to_message(self, order, message):
...
         return message
 
+    def _submit_fluffy(self, order: dict) -> dict:
+        """Submit a batch of ready made libraries for FLUFFY analysis."""
+        return self._submit_pools(order)
+
     def _submit_rml(self, order: dict) -> dict:
-        """Submit a batch of ready made libraries."""
+        """Submit a batch of ready made libraries for sequencing."""
+        return self._submit_pools(order)
```
Comment (Mropat) [inline cg/meta/orders/api.py:182]: What is the benefit of splitting submit rml and submit fluffy off pools? Seems they just pass the method call forward without any unique handling

Comment (patrikgrenfeldt) [reply cg/meta/orders/api.py:182]: yeah, it's a dynamic call to the method, I guess we could have solved it by using the get-method as well

**Comment by patrikgrenfeldt** [comment]
Deployed to clinical-api:
image

**Pull request merged**